vsc_pdf_template_editor/lib/src/stores/tree_store.dart

```diff
@@ -239,6 +243,9 @@ abstract class TreeStoreModel with Store {
       case 'Limited Box':
         map = TplLimitedBox().toJson();
         break;
+      case 'Image':
+        map = TplImage(TplMemoryImage(imageData)).toJson();
```

Comment:
As I mentioned in the editor, this one should just add `TplImage().toJson()`.  The "Add image" button will add a TplMemoryImage